hadoop/src/main/java/org/apache/carbondata/hadoop/streaming/CarbonStreamRecordWriter.java

```diff
@@ -173,6 +172,8 @@ private void initializeAtFirstRow() throws IOException, InterruptedException {
       outputStream = FileFactory.getDataOutputStream(filePath, fileType);
       writeFileHeader();
     }
+
+    isFirstRow = false;
```

Comment:
Please modify this in another PR

Reply:
it means whether current row is first row or not